src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -43,7 +43,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' <summary> 
         ''' The root node of the syntax tree that this binding is based on.
         ''' </summary> 
-        Friend MustOverride ReadOnly Property Root As SyntaxNode
+        Friend MustOverride Shadows ReadOnly Property Root As SyntaxNode
```

Comment:
It's strange that this doesn't return the VB-specific type (VisualBasicSyntaxNode).
Filed [https://github.com/dotnet/roslyn/issues/21266](https://github.com/dotnet/roslyn/issues/21266) to track that.